src/go/rpk/pkg/cli/cluster/partitions/move.go

```diff
@@ -0,0 +1,176 @@
+package partitions
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newMovePartitionReplicasCommand(fs afero.Fs, p *config.Params) *cobra.Command {
+	var partitions []string
+	cmd := &cobra.Command{
+		Use:   "move",
+		Short: "Move partition replicas across nodes / cores",
+		Long:  helpAlterAssignments,
+		Run: func(cmd *cobra.Command, topics []string) {
+			p, err := p.LoadVirtualProfile(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+			out.CheckExitCloudAdmin(p)
+
+			cl, err := adminapi.NewClient(fs, p)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			for i, topic := range topics {
+				if !strings.Contains(topic, "/") {
+					topics[i] = "kafka/" + topic
+				}
+			}
+			for _, partition := range partitions {
+				if len(topics) > 0 { // foo -p 0:1,2,3
+					p, replicas, err := parsePartitionReplicas(partition)
+					out.MaybeDie(err, "unable to execute the request: %v\n", err)
+					for _, nt := range topics {
+						err = cl.MoveReplicas(cmd.Context(), nt+"/"+p, replicas)
+						out.MaybeDie(err, "unable to move partition replicas: %v\n", err)
+					}
```

Comment:
Can this range be concurrent? I think we use errgroup in rpk somewhere; use that.

Reply:
Addressed in https://github.com/redpanda-data/redpanda/commit/1595272ed458ac47621cdbcad4c550c99140d622